irobotclient/test/test_main.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from unittest.mock import MagicMock
+
+import requests
+import os
+
+import irobotclient.main
+
+
+class TestMain(unittest.TestCase):
+
+    test_data_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'testdata/')
+    output_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'testdir/')
+
+    def setUp(self):
+        self._old_request_head = requests.head
+        self._response = requests.Response()
+        requests.head = MagicMock(spec=requests.head)
```

Comment:
Can you elaborate @colin-nolan? Because I don't have a working web server I thought the only way to mimic irobot responses was to mock the requests methods to return a custom response, unless I use HoverFly maybe...